testsuite/classlibrary/TestCorrectArgCount.sc

```diff
@@ -0,0 +1,12 @@
+TestCorrectArgCountObj {
+	var <a, <b, <c;
+	*new {|a| ^super.newCopyArgs(a) }
+}
+TestCorrectArgCount : UnitTest {
+	test_correct_arg {
+		var o = TestCorrectArgCountObj(\a, \b, \c);
```

Comment:
better add messages:

this.assertEquals(o.a, \a, "first argument given in the method definition is passed correctly");
		this.assertEquals(o.b, nil, "arguments absent from method definition should not be passed");
		this.assertEquals(o.c, nil, "arguments absent from method definition should not be passed");